pkg/prowloader/prow.go

```diff
@@ -136,6 +145,30 @@ func (pl *ProwLoader) LoadProwJobsToDB() error {
 	return nil
 }
 
+func (pl *ProwLoader) syncPRStatus() error {
+	if pl.githubClient == nil {
+		log.Infof("No GitHub client, skipping PR sync")
+		return nil
+	}
+
+	pulls := make([]models.ProwPullRequest, 0)
+	pl.dbc.DB.Table("prow_pull_requests").Where("merged_at IS NULL").Scan(&pulls)
+	for _, pr := range pulls {
+		mergedAt, err := pl.githubClient.GetPRMerged(pr.Org, pr.Repo, pr.Number, pr.SHA)
+		if err != nil {
+			log.WithError(err).Warningf("could not fetch pull request status from GitHub; org=%q repo=%q number=%q sha=%q", pr.Org, pr.Repo, pr.Number, pr.SHA)
+			return err
+		}
+		pr.MergedAt = mergedAt
```

Comment:
Should we check if it changed here before doing the Save to limit writes?

Reply:
Good call, fixed